eej_prior_commits.txt

```diff
@@ -0,0 +1,26 @@
+The following commits were made pursuant to the Developer Certificate of Origin, even though a Signed-off-by: was not included in the commit message.
```

Comment:
Could this file be placed inside a "dco_signoffs" folder?

Sorry for being picky 😢 I'd prefer the number of files at the top level to be as few as possible and have noticed that other Zowe repos use a subfolder for their DCO signoffs ([docs site](https://github.com/zowe/docs-site/tree/master/dco_signoffs), [Zowe Explorer](https://github.com/zowe/vscode-extension-for-zowe/tree/master/dco_signoffs)).

Reply:
I completely agree. I did not like it when I did it, but I obediently followed the instructions that I found.
However, this is our repo, so we should be able to do what makes sense to us.
I will move the file into a new dco_signoffs folder.